doc/build/changelog/unreleased_20/9752.rst

```diff
@@ -0,0 +1,14 @@
+.. change::
+    :tags: usecase, sql
+    :tickets: 9752
+
+
+    Generalized the MSSQL :func:`.try_cast` function to any dialect.
```

Comment:
I think it would make sense to mention that even after this change of the internal dialect only MSSQL supports it

Reply:
done